certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Document AI Konfuzio",
+    "termsOfService": "https://konfuzio.com/impressum/",
+    "contact": {
+      "name": "Helm & Nagel GmbH",
+      "email": "info@konfuzio.com",
+      "url": "https://help.konfuzio.com"
+    },
+    "license": {
+      "name": "Helm & Nagel GmbH"
+    },
+    "version": "v2",
```

Comment:
@atraining This is a version of your connector. Since this is a new connector, it can be 

`"version": "1.0.0"`

Reply:
Hi @sriyen-msft thanks for pointing this out. I changed the version to 1.0.0